<ui version="4.0">
 <author/>
 <comment/>
 <exportmacro/>
 <class>editor::Slider_Bar</class>
 <widget class="QWidget" name="editor::Slider_Bar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="rank_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rank_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="max_rank_label">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok_button">
        <property name="text">
         <string>&amp;Ok</string>
        </property>
       </widget>
      </item>
     </layout>
 </widget>
 <pixmapfunction/>
 <connections/>
</ui>
